tests/verify_changes.py

```
@@ -0,0 +1,134 @@
+        else:
+            print("⚠️ No interactions found (might be filtered out or failed).")
+            
+    except Exception as e:
+        print(f"❌ Crash during interaction test: {e}")
+        import traceback
+        traceback.print_exc()
+
+def test_poisson_basic(df):
+    print("\n[TEST 3] Poisson Regression Basic")
+    try:
+        html, irr_res, airr_res, int_res = analyze_poisson_outcome(
```

Ruff (0.14.10)

Unpacked variable `html` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)